### DISCUSSION TITLE
Alternate Use Case with Wahoo Kickr Snap

### DISCUSSION THREAD

**Discussion by ibmcphers** [cagnulein/qdomyos-zwift#1160] [Q&A]
Hi,

Does anyone know if this app work with a smart trainer like the Wahoo Kickr Snap?

I currently have a lot of issues with the Zwift Companion app running on Android when paired with Apple TV in addition to terrible support from Zwift.  After a brief read, it looks like this app would perform the same sensor aggregation so that I could overcome the Apple TV 2 BLE channel limitation and not need to use the companion app for sensor pairing.

Would the trainer resistance messages from Zwift to the Wahoo be transmitted via the app to allow the simulation of hills or ERG mode?

Thanks for any insight.

Ian

**Comment by cagnulein** [comment]
hi @ibmcphers yes! it's already possible! i would like to do a video about the fact that qz can also remove the apple watch limitation of the 2 bluetooth devices!

Give it a try and let me know if you need anything else!

**Reply by ibmcphers** [reply]
Hi @cagnulein 

I purchased the app and gave it a try.  When given the option to pair with the Apple TV there were a couple of options presented - my Pixel 7 over Bluetooth or it will show the Wahoo Kickr over a physical Ethernet.  Either option does not allow for in game calibration.  Not sure what the difference is in the presented connection method.  I didn't ride for long, it was a quick test but it seemed to be working.

Ian

**Reply by cagnulein** [reply]
calibration is not needed :) because all the metrics are directly got from your kickr. Glad to hear it!

**Reply by ibmcphers** [reply]
BTW, very cool app and I realize I'm trying to use it outside of its standard application.  The other mode I tried was to see if I could keep the kickr directly connected to the ATV and only aggregate the HR and Cadence sensors (to get over the 2 BLE channel limit) but really that defeats the primary purpose for the app, so I see why that isn't a possibility.

Ian

**Reply by cagnulein** [reply]
yes you can do setting the cadence sensor in the qz app and also enabling cadence sensor as a bike setting
